miden-remote-provers/Cargo.toml

```diff
@@ -0,0 +1,46 @@
+[package]
+name = "miden-remote-provers"
+version = "0.1.0"
+description = "client library that provides a client for the remote provers"
```

Comment:
nit: Is it worth changing the name to `miden-remote-provers-client`? Maybe it's too much of a mouthful but it sounds more descriptive at a first glance (`miden-remote-provers` sounds more ambiguous in comparison). cc @bobbinth since I think he suggested these names

Reply:
I'm not a big fan of `miden-remote-prover-clients`, maybe something like `miden-prover-clients`?

Reply:
`miden-prover-client` may be fine, but also maybe `miden-proving-service-client` is better? (it immediately indicates that this is a "companion" crate for `miden-proving-service` crate).